elements/bulbs-video/components/revealed.js

```diff
@@ -156,13 +160,54 @@ export default class Revealed extends React.Component {
     return false;
   }
 
-  vastUrl (videoMeta) {
-    let baseUrl = 'http://us-theonion.videoplaza.tv/proxy/distributor/v2?rt=vast_2.0';
+  getProfValue () {
+    if (window.isMobile.any) {
+      return 'theonion_mobileweb_html5';
+    } else {
+      return 'theonion_desktop_html5';
+    }
+  }
+
+  setDeviceAcronym () {
+    if (window.isMobile.any) {
+      return 'm';
+    } else {
+      return 'd';
+    }
+  }
+
+  getSiteName () {
+    return window.location.host.split('.')[1];
```

Comment:
some videos have their `target_channel` set to `onion-studios`, do we want that value or should I just make it default to theonion?